Commands/ConfigPanelCommand.cs

```diff
@@ -0,0 +1,24 @@
+using ChestContents.Managers;
+using Jotunn.Entities;
+
+namespace ChestContents.Commands
+{
+    public class ConfigPanelCommand : ConsoleCommand
+    {
+        public override string Name => "chestconfig";
+        public override string Help => "Open the ChestContents config panel";
+        public override bool IsNetwork => false;
+
+        public override void Run(string[] args)
+        {
+            ChestContentsPlugin.Logger.LogWarning("/chestconfig command run.");
+            if (ChestContentsPlugin.ConfigPanelManagerInstance == null)
+            {
+                ChestContentsPlugin.Logger.LogWarning("Config panel is not available yet. Try again in a few seconds after entering the world.");
```

Comment:
This is probably worth removing if all cases of the instance is around when expected.